azext_iot/iothub/_help.py

```diff
@@ -309,3 +309,43 @@ def load_iothub_help():
         type: command
         short-summary: Upload a local file as a device to a pre-configured blob storage container.
     """
+
+    helps["iot edge hierarchy"] = """
```

Comment:
We discussed the command name change making sure there is ample examples of hierarchy based + non-hierarchy based edge device configuration examples.